### PR TITLE
MGMT-7919 - Filter out OCP versions with maintenance support level in the create cluster screen

### PR DESCRIPTION
**Pull request by jgyselov**
https://issues.redhat.com/browse/MGMT-7919

### CODE REVIEW

File: src/common/api/types.ts [hunk 1]
```diff
@@ -1723,7 +1723,7 @@ export interface OpenshiftVersion {
   /**
    * Level of support of the version.
    */
-  supportLevel: 'beta' | 'production' | 'custom';
+  supportLevel: 'beta' | 'production' | 'custom' | 'maintenance';
```
Comment (jkilzi) [inline src/common/api/types.ts:1726]: is this coming from swagger? I don't see an updated swagger.yaml file here?

Comment (jkilzi) [reply src/common/api/types.ts:1726]: this file should not be modified manually

Comment (jkilzi) [reply src/common/api/types.ts:1726]: seems like `custom` is not part of this interface anymore
image

Comment (jgyselov) [reply src/common/api/types.ts:1726]: No, it's added manually. Should I update the swagger.yaml instead? @jkilzi

Comment (jkilzi) [reply src/common/api/types.ts:1726]: yes please

**Review by jkilzi — APPROVED** [review]

**Pull request merged**